ur_description/urdf/ur5_robot.urdf.xacro

```diff
@@ -5,6 +5,27 @@
   <!-- ur5 -->
   <xacro:include filename="$(find ur_description)/urdf/ur5.urdf.xacro" />
 
+  <!-- ros2 control argument-->
+  <xacro:arg name="use_ros2_control" default="true"/>
```

Comment:
Can you use it without?

Reply:
fake_joint_driver could be instantiated aside.